### PR TITLE
Gidk 637 dev | Rohit | Loader is implemented for login signup

### PR DESCRIPTION
**Pull request by rohit-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/login/login.component.ts [hunk 1]
```diff
@@ -153,7 +154,9 @@ export class LoginComponent implements OnInit, OnDestroy {
             return state.login.isLoginWithPasswordIsShowVerifyOtp;
         }).pipe(takeUntil(this.destroyed$));
         this.isSocialLogoutAttempted$ = this.store.select(p => p.login.isSocialLogoutAttempted).pipe(takeUntil(this.destroyed$));
-
+        this.isLoginWithGoogleInProcess$ = store.pipe(select(state => {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/login/login.component.ts:157]: Use this with store
```suggestion
        this.isLoginWithGoogleInProcess$ = this.store.pipe(select(state => {
```

File: apps/web-giddh/src/app/signup/signup.component.ts [hunk 2]
```diff
@@ -74,24 +74,26 @@ export class SignupComponent implements OnInit, OnDestroy {
...
+        private router: Router,
+        private loginAction: LoginActions,
+        private authService: AuthService,
+        @Inject(DOCUMENT) private document: Document,
+        private _toaster: ToasterService,
+        private _generalService: GeneralService
     ) {
         this.urlPath = (isElectron || isCordova) ? "" : AppUrl + APP_FOLDER;
-        this.isLoginWithEmailInProcess$ = store.select(state => {
+        this.isLoginWithEmailInProcess$ = store.pipe(select(state => {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/signup/signup.component.ts:91]: Use `this` with store

File: apps/web-giddh/src/app/signup/signup.component.ts [hunk 3]
```diff
@@ -74,24 +74,26 @@ export class SignupComponent implements OnInit, OnDestroy {
...
             return state.login.isLoginWithEmailInProcess;
-        }).pipe(takeUntil(this.destroyed$));
-        this.isVerifyEmailInProcess$ = store.select(state => {
+        }), takeUntil(this.destroyed$));
+        this.isVerifyEmailInProcess$ = store.pipe(select(state => {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/signup/signup.component.ts:94]: Use `this` with store

File: apps/web-giddh/src/app/signup/signup.component.ts [hunk 4]
```diff
@@ -126,13 +128,15 @@ export class SignupComponent implements OnInit, OnDestroy {
         this.isSignupWithPasswordSuccess$ = store.select(state => {
             return state.login.isSignupWithPasswordSuccess;
         }).pipe(takeUntil(this.destroyed$));
-
+        this.isLoginWithGoogleInProcess$ = store.pipe(select(state => {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/signup/signup.component.ts:131]: Use `this` with store

File: apps/web-giddh/src/app/login/login.component.ts (hunk 1 above)
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/login/login.component.ts:157]: use this with store object

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.html [hunk 5]
```diff
@@ -368,12 +368,12 @@ <h3 class="font-20 bold mb-2">Amazon Seller</h3>
                     </div>
                 </div>
             </div>
-            <!-- <div class="mrT2 box clearfix">
+            <div class="mrT2 box clearfix">
                 <h3 class="section-head mb-2">Shopify</h3>
                 <span *ngIf="isEcommerceShopifyUserVerified">Your company is successfully connected with Shopify.</span>
                 <a href="https://apps.shopify.com/accounting-software" target="_blank" *ngIf="!isEcommerceShopifyUserVerified">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/settings/integration/setting.integration.component.html:374]: Please update the url https://shopify.giddh.com/signup.php

File: apps/web-giddh/src/app/signup/signup.component.ts (hunk 2 above)
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/signup/signup.component.ts:91]: use this with store object

File: apps/web-giddh/src/app/store/authentication/authentication.reducer.ts [hunk 6]
```diff
@@ -381,6 +388,10 @@ export function AuthenticationReducer(state: AuthenticationState = initialState,
                         user: res.body
                     });
                 }
+            } else {
+              return  Object.assign({}, state, {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/store/authentication/authentication.reducer.ts:392]: use formatting

**Pull request merged**